public/app/features/alerting/unified/previewToggles.ts

```diff
@@ -0,0 +1,28 @@
+import { FeatureToggles, Store } from '@grafana/data';
+
+type AlertingPreviewToggles = Pick<FeatureToggles, 'alertingListViewV2'>;
+
+const defaultPreviewToggles: AlertingPreviewToggles = {
+  alertingListViewV2: undefined,
+};
+
+const previewToggleStoreKey = 'grafana.alerting.previewToggles';
+
+/**
+ * Get the preview toggle value for the given toggle name.
+ * @returns The value of the preview toggle or undefined if it is not set.
+ */
+export function getPreviewToggle(previewToggleName: keyof AlertingPreviewToggles): boolean | undefined {
+  const previewStore = new Store();
```

Comment:
why do you use a new instance of Store? is not enough the singleton store from `grafana-data` ?

Reply:
I haven't noticed it 😅 I'll use the store singleton